src/ripper.rb

```diff
@@ -39,9 +39,7 @@ def on_assoc_new(left, right)
   end
 
   def on_bodystmt(*body)
-    build_sexp(:bodystmt, body).tap do |sexp|
```

Comment:
Separate PR for formatting this file: https://github.com/prettier/plugin-ruby/pull/123